docs/docs/60-new-docs/50-user-guide/40-examples.md

```diff
@@ -4,8 +4,30 @@ sidebar_label: Examples
 
 # Examples
 
-:::warning
-This section is expected to be quite extensive.
+Here you'll find practical demonstrations and reference implementations showcasing how to effectively use Kargo in your GitOps workflows.
+
+Whether you're new to Kargo or looking to optimize your existing deployment pipelines, these examples will help you understand key concepts and implement powerful CD workflows. Each example includes detailed explanations, configuration files, and step-by-step guides to get you started.
+
+## Basic
+### Argo CD-Driven Examples
+* [Argo CD Driven | Commit Only](https://github.com/akuity/kargo-examples/tree/main/01-argocd-driven/01-commit-only)
+* [Argo CD Driven | Helm Driven | Chart Only](https://github.com/akuity/kargo-examples/tree/main/01-argocd-driven/02-helm-driven/01-chart-only)
+* [Argo CD Driven | Helm Driven | Image only + Chart Repo](https://github.com/akuity/kargo-examples/tree/main/01-argocd-driven/02-helm-driven/02-image-only/01-with-chart-repo)
+* [Argo CD Driven | Helm Driven | Image only + Git Repo](https://github.com/akuity/kargo-examples/tree/main/01-argocd-driven/02-helm-driven/02-image-only/02-with-git-repo)
+* [Argo CD Driven | Helm Driven | Chart and Image](https://github.com/akuity/kargo-examples/tree/main/01-argocd-driven/02-helm-driven/03-chart-n-image)
+* [Argo CD Driven | Helm Driven | Commit + Image](https://github.com/akuity/kargo-examples/tree/main/01-argocd-driven/02-helm-driven/04-commit-n-image) 
+* [Argo CD Driven | Kustomize Driven | Image only](https://github.com/akuity/kargo-examples/tree/main/01-argocd-driven/03-kustomize-driven/01-image-only)
+* [Argo CD Driven | Kustomize Driven | Commit + Image](https://github.com/akuity/kargo-examples/tree/main/01-argocd-driven/03-kustomize-driven/02-commit-n-image)
+
+### Git Driven
+* [Git Driven | Commit Only](https://github.com/akuity/kargo-examples/tree/main/02-git-driven/01-commit-only)
+* [Git Driven | Helm Driven | Image Only](https://github.com/akuity/kargo-examples/tree/main/02-git-driven/02-helm-driven/01-image-only)
+* [Git Driven | Helm Driven | Commit + Image](https://github.com/akuity/kargo-examples/tree/main/02-git-driven/02-helm-driven/02-commit-n-image)
+* [Git Driven | Kustomize Driven | Image Only](https://github.com/akuity/kargo-examples/tree/main/02-git-driven/03-kustomize-driven/01-image-only)
+* [Git Driven | Kustomize Driven | Commit + Image](https://github.com/akuity/kargo-examples/tree/main/02-git-driven/03-kustomize-driven/02-commit-n-image) 
+
+## Advanced
+* [kargo-simple](https://github.com/akuity/kargo-simple)
```

Comment:
Is it strange for an example with "simple" in its name to appear under the heading of "Advanced?"